### DISCUSSION TITLE
将alist挂载为webdav使用时，本地更新文件后，alist网页端列表不是更新替换文件而是会出现很多相同的文件

### DISCUSSION THREAD

**Discussion by idmanage** [AlistGo/alist#695] [Q&A]
将alist挂载为webdav使用时，本地更新文件后，alist网页端列表不是更新替换文件而是会出现很多相同的文件
如：本地有新建一个文件123.txt，alist网页端列表也会出现一个新建的文件123.txt，但是本地修改这个123.txt文件保存后，alist网页端列表便会又新建一个123.txt文件，试了很多次，只要修改保存，alist网页端列表就会重新新建一个123.txt文件，最后alist网页端列表和网盘内会出现很多重名的123.txt，请大佬修复BUG，谢谢
还有一个问题，alist网页端列表假如存在6个相同的123.txt，全选这6个并删除，发现网盘和alist网页端列表其实只删除了1个，需要删除6次，才能彻底删除

**Comment by xhofe** [comment]
什么存储。

**Reply by idmanage** [reply]
Teambition，也试过阿里云盘和百度网盘，也是一样的问题

**Reply by idmanage** [reply]
我觉得在更新文件时，先检查是否存在，存在的话先删除再复制文件

**Reply by FrozenSky7124** [reply]
大佬，我和楼主的问题挺像的，Teambition存放了一个视频V，我在win本地挂载上webdav文件夹，本地播放视频的时候发现文件夹里又多了两个视频文件：V(1), V(2)

**Reply by xhofe** [reply]
因为teambition 不支持覆盖上传，且程序无法保证上传成功。所以不敢做`先检查是否存在，存在的话先删除再复制文件`这种

**Reply by zsbai** [reply]
遇到过类似的问题，谷歌云盘和onedrive均出现过出现大量重复文件夹的情况，当时排查还以为是jellyfin的问题233
但是阿里云盘没有

**Reply by idmanage** [reply]
阿里云盘也有问题，虽然不出现重复的文件，但是会出现1.txt，1(1).txt，1(2).txt这种文件
